src/__tests__/App.test.js

```diff
@@ -0,0 +1,18 @@
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import store from '../redux/configureStore';
+import App from '../App';
+
+describe('matches snapshot', () => {
+  test('matches to snapshot', () => {
+    const tree = renderer.create(
+      <Router>
+        <Provider store={store}>
+          <App />
+        </Provider>
+      </Router>,
+    ).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
Well done adding unit tests for important components of the app. Great job!!! 💯 

There are a few more tests that are missing.
As requested by the previous reviewer please add unit tests for pure functions in your Redux code and integration tests for your application using the React Testing Library.

This means:

- Add unit tests for some pure functions in Redux code (these are your action creators). 
- You can perform a mocking data fetching and Redux Store connection to add integration tests using React Testing Library.